packages/graphql-live-query/src/GraphQLLiveDirective.ts

```diff
@@ -11,5 +11,9 @@ export const GraphQLLiveDirective = new GraphQLDirective({
       defaultValue: true,
       description: "Whether the query should be live or not.",
     },
+    throttle: {
+      type: GraphQLInt,
+      description: "Limit updates to at most once per \"throttle\" milliseconds."
+    }
```

Comment:
I like the idea that the operation execution initiator can specify the throttle level 🔥  As not every engine MUST implement a throttle, we should probably change the wording to something like `Give a hint to the server on the rate of updates you would wish to receive`.